paper/paper.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Pyglmnet: Python implementation of elastic-net regularized generalized linear models'
+tags:
+  - Python
+  - glm
+  - machine-learning
+  - lasso
+  - elastic-net
+  - group-lasso
+authors:
+  - name: Mainak Jas
+    orcid: 0000-0003-0872-7098
+    affiliation: "1, 2"
+  - name: Titipat Achakulvisut
+    affiliation: 3
+  - name: Aid Idrizović
+    affiliation: 4
+  - name: Daniel Acuna
+    affiliation: 5
+  - name: Matthew Antalek
+    affiliation: 6
+  - name: Vinicius Marques
+    affiliation: 4
+  - name: Tommy Odland
+    affiliation: 7
+  - name: Ravi Prakash Garg
+    affiliation: 6
+  - name: Mayank Agrawal
+    affiliation: 8
+  - name: Yu Umegaki
+    affiliation: 9
+  - name: Peter Foley
+    affiliation: 10
+  - name: Hugo Fernandes
+    affiliation: 11
+  - name: Drew Harris
+    affiliation: 12
+  - name: Beibin Li
+    affiliation: 13
+  - name: Olivier Pieters
+    affiliation: 14
+  - name: Scott Otterson
+    affiliation: 15
+  - name: Giovanni De Toni
```

Comment:
My name and affiliation are okay! Regarding the orcid, here it is the code comment: 
```suggestion
  - name: Giovanni De Toni
  - orcid: 0000-0002-8387-9983
```